package.json

```diff
@@ -5,6 +5,7 @@
     "prettier": "2.3.2"
   },
   "dependencies": {
+    "@fontsource/quicksand": "^4.5.4",
```

Comment:
hmm, why is this added in the root package.json? all the easter fonts should be already be imported. maybe do a pull --rebase from easter branch?